dotcom-rendering/src/components/SlotBodyEnd.importable.tsx

```diff
@@ -124,9 +124,9 @@ export const SlotBodyEnd = ({
 	const [countryCode, setCountryCode] = useState<string>();
 	const isSignedIn = getIsSignedIn(useAuthStatus());
 	const browserId = getCookie({ name: 'bwid', shouldMemoize: true });
-	const [SelectedEpic, setSelectedEpic] = useState<React.ElementType | null>(
-		null,
-	);
+	const [SelectedEpic, setSelectedEpic] = useState<
```

Comment:
```suggestion
	const [selectedEpic, setSelectedEpic] = useState<
```

Reply:
It's a [component](https://github.com/guardian/dotcom-rendering/pull/9180/files#diff-2a3e6c846cfb3ce97cafc0e87288e340ce3ec187fe7933525c58c3948935b670L206) so I think it should be capitalised?

Reply:
Gotcha. Am I right in thinking it's starts and `undefined` then gets updated to `null` if there isn't an epic?

Reply:
Yep, I added the `undefined` state so that the custom event will only be sent when `SelectedEpic` has been specifically set to `null`, also stops the ad was flickering into view while the Epic decides what it's doing.